docs/overlay-hld.adoc

```diff
@@ -30,6 +30,9 @@
  |Ofer Shinaar
  |0.5 |May 2,2021| pdf fixups formatting for pdf
  |Ofer Shinaar
+ |0.6 |May 19,2021| add comments for: a)This design support only rv32 arch
+ b)The design does not support overlay ISR's
+ |Ofer Shinaar
```

Comment:
The commit message suggests the changes are all about ISRs. Can you mention in the commit message that you are also documenting the supported XLEN and required extensions?